repair/row_level.cc

```diff
@@ -646,6 +649,13 @@ future<std::list<repair_row>> to_repair_rows_list(repair_rows_on_wire rows, sche
         }
         co_await coroutine::maybe_yield();
     }
+  } catch (...) {
+    ex = std::current_exception();
+  }
+    if (ex) {
+        co_await utils::clear_gently(row_list);
```

Comment:
You can also clear rows here. If the exception happened early most of the data would be there.

Reply:
good idea